integration-cli/docker_cli_run_test.go

```diff
@@ -144,18 +144,18 @@ func TestRunLeakyFileDescriptors(t *testing.T) {
 	logDone("run - check file descriptor leakage")
 }
 
-// it should be possible to ping Google DNS resolver
+// it should be possible to lookup Google DNS
 // this will fail when Internet access is unavailable
-func TestRunPingGoogle(t *testing.T) {
+func TestRunLookupGoogleDns(t *testing.T) {
 	defer deleteAllContainers()
 
-	runCmd := exec.Command(dockerBinary, "run", "busybox", "ping", "-c", "1", "8.8.8.8")
-	out, _, _, err := runCommandWithStdoutStderr(runCmd)
+	runCmd := exec.Command(dockerBinary, "run", "busybox", "sh", "-c", `[[ -n "$(nslookup 8.8.8.8 | grep google.com)" ]]`)
```

Comment:
@icecrime because "nslookup 8.8.8.8" doesnt exit with a nonzero code even when network is not present. It's still successful but doesn't contain "google.com" string in the output. @tianon